### PR TITLE
update chat workshop

### PR DESCRIPTION
**Pull request by aarohmankad**
working on #912, need to finish Part V before this can be merged.

Look over what I have so far? @MaxWofford

### CODE REVIEW

**Comment by aarohmankad** [comment]
I still have to create a gif of the final product too (that's why it's not passing the tests)

File: workshops/chat/README.md [hunk 1]
```diff
@@ -301,7 +320,7 @@ Congratulations. Your chat app is complete. Save all your files and commit your
 - `git commit -m "Create a chat app using Firebase"`
 - `git push`
 
-**After entering your username and password, you'll need to add `USERNAME.github.io` to the list of Authorized Domains in Firebase.**
+**After entering your username and password, you will need to add `USERNAME.github.io` to the list of Authorized Domains in Firebase.** You can see how we did this earlier in [Configuring Permissions](#configuring-permissions).
```
Comment (MaxWofford) [inline workshops/chat/README.md]: What if we do this in the [Configuring Permissions](#configuring-permissions) step so hackers don't have to go back later?

Comment (aarohmankad) [inline workshops/chat/README.md]: ah good idea. I'll change it with the next commit

**Comment by MaxWofford** [comment]
@aarohmankad I'm assigning you while the PR is still in progress. Feel free to assign me once it's ready for review.

**Comment by zachlatta** [comment]
Hope you don't mind that I went ahead and merged this, seemed like it was all good.

**Pull request merged**